frontend/test/storybook/utils/response.ts

```diff
@@ -0,0 +1,18 @@
+import { type Page } from "@playwright/test"
+
+/**
+ * Wait for a response to a URL that matches a regex.
+ * The pattern of setting up a wait for response without `await`,
+ * and awaiting the response after navigating to the URL is used
+ * to prevent race conditions.
+ * @see https://playwright.dev/docs/api/class-page#page-wait-for-request.
```

Comment:
Very detailed explanations of the race conditions: https://github.com/microsoft/playwright/issues/5470#issuecomment-1285640689
They do offer `Promise.all` for this, but later Playwright docs moved to this setup.